beetmoverscript/script.py

```diff
@@ -43,14 +46,18 @@
     #   a. download artifact
     #   b. upload to candidates/dated location
     await move_beets(context, manifest)
-
+    # 6. write balrog_manifest to a file and add it to list of artifacts
+    if context.task["payload"]["update_manifest"]:
```

Comment:
In the attempt to ditch `update_manifest` variable, here we could substitue with something like:

if context.balrog_manifest:
         add_balrog_manifest_to_artifacts(context)

since `balrog_manifest` will remain an empty list if no task from manifest has the  `update_balrog_manifest` flag set. However, we'd still need to use the `update_manifest` var when we toggle between the unsigned/signed templates.